contrib/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/grafana/pyroscope-go/contrib
```

Comment:
Should we go further and put `k6` stuff into a submodule? `contrib` would be just a container for integration sub-modules.

Otherwise, once we add another integration, its dependencies would be added to the k6 package, and vice-versa

Reply:
Yep! I agree with this move! I was attempting to do that last Friday but `go work sync` was fighting with me. It kept trying to update the go version of other modules.

I'll continue to work on this though and get it addressed before merging.

Reply:
Okay, I got this fixed in https://github.com/grafana/pyroscope-go/pull/99/files/cbda95a5bdb83f75d53d811f4b921385e53e6ed0..016a1717a57831f63723fc46f4298531c17ac3f5. A couple things to note:

* I marked the `k6` package as experimental by moving it to the `x/k6` submodule because this API  may change or be removed entirely at any time.
* I had to downgrade the go.opentelemetry.io/otel package from `1.27.0` to `1.17.0` to make the submodule compatible with go versions <1.21.